NEWS

```diff
@@ -8,6 +8,30 @@ Removals:
     names `mongoc.h` or `bson.h` directly. Instead, the names must be prefixed
     with the parent directory: `mongoc/mongoc.h` and `bson/bson.h`.
 
+Changes:
+
+  * Authentication mechanisms and connection strings:
+    * Case sensitive is now respected for _values_ of authentication-related URI options and authentication mechanism properties.
+      * _Keys_ remain case-insensitive, including in mechanism properties. e.g. `authMechanismProperties=service_name:value` is equivalent to `authMechanismProperties=SERVICE_NAME:value`.
+      * e.g. `"authMechanism=scram-sha-1"` does not equal `"authMechanism=SCRAM-SHA-1"`.
```

Comment:
Suggest preserving the existing case-insensitive comparison to reduce the risk of breaking applications upgrading to 2.0.

There may be precedent with other products. Using mongosh (WireShark indicates SCRAM-SHA-1 is used):
```
% mongosh "mongodb://user:password@localhost/?authMechanism=scram-sha-1" --eval 'db.runCommand({"ping": 1})' --quiet
```